src/storage/storage.cc

```diff
@@ -183,6 +183,29 @@ rocksdb::Options Storage::InitRocksDBOptions() {
 }
 
 Status Storage::SetOptionForAllColumnFamilies(const std::string &key, const std::string &value) {
+  if ("compression" == key) {
+    std::unordered_map<std::string, std::string> compression_map = {{"no", "kNoCompression"},
```

Comment:
It'd be better to put this text in compression_types, to make it more easy to maintain. Or it needs to take care of many places if we to add a new compression type.

Reply:
> It'd be better to put this text in compression_types, to make it more easy to maintain. Or it needs to take care of many places if we wanna add a new compression type.

done